components/commons/overlay-coretokens.css

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+%spectrum-overlay {
+  /* TODO: replace with core tokens */
+  --spectrum-overlay-animation-distance: 6px;
+  --spectrum-global-animation-duration-100: 130ms;
+
+  visibility: hidden;
+
+  opacity: 0;
+
+  transition: transform var(--spectrum-global-animation-duration-100) ease-in-out,
+              opacity var(--spectrum-global-animation-duration-100) ease-in-out,
+              visibility 0ms linear var(--spectrum-global-animation-duration-100);
+
+  pointer-events: none;
+}
+
+%spectrum-overlay--open {
+  visibility: visible;
+  opacity: 1;
+  transition-delay: 0ms;
+  pointer-events: auto;
+}
+
+%spectrum-overlay--bottom--open {
```

Comment:
Is this an existing naming convention with e.g. `--bottom--open`?

Reply:
Sorry, I don't understand the question. This mixin already existed in overlay. I did not create any new mixins for popover.

Reply:
Ok, no problem. I was just curious because it looked like a double BEM modifier.